services/mongoose.js

```diff
@@ -40,14 +41,23 @@ if (enabled('talk:db')) {
   mongoose.set('debug', debugQuery);
 }
 
-// Connect to the Mongo instance.
-mongoose.connect(MONGO_URL, (err) => {
-  if (err) {
-    throw err;
-  }
+if (WEBPACK) {
+
+  console.warn('Not connecting to mongodb during webpack build');
+  mongoose.disconnect();
```

Comment:
Not exactly. We don't call connect, but because we include it, it will hold the socket ready, preventing node from exiting. Calling disconnect here just ensures that the application can quit correctly.